src/group.rs

```diff
@@ -632,12 +626,14 @@ impl<'a> GroupMutItem<'a> {
         match self.entry_exact_mut(entry_id, instance_id, board_instance_mask) {
             Some(e) => Ok(e),
             None => {
-                panic!("Entry (entry_id = {:?}, instance_id = {:?}, board_instance_mask = {:?}) was found by move_point to, but not by manual iteration after resizing.", entry_id, instance_id, board_instance_mask);
+                panic!(
+                    "Entry (entry_id = {:?}, instance_id = {:?}, board_instance_mask = {:?}) was found by move_point to, but not by manual iteration after resizing.",
+                    entry_id, instance_id, board_instance_mask
+                );
             }
         }
     }
     /// Inserts the given token.
-    #[pre("Caller already grew the group by `size_of::<TOKEN_ENTRY>()`")]
```

Comment:
Please make this a "/// Precondition:" comment or something.

Reply:
Done.